app/controllers/listings_controller.rb

```diff
@@ -484,6 +498,22 @@ def verification_required
 
   private
 
+  def create_bookable(listing_uuid)
+    res = HarmonyClient.post(
+      :create_bookable,
+      body: {
+        marketplaceId: @current_community.uuid,
+        refId: listing_uuid,
+        authorId: @current_user.uuid
+      })
```

Comment:
One principle we've tried to keep with controllers is that only the top level method, i.e. the action, pulls data from global scope and all delegated to methods get things like marketplace and auhor as parameters. This makes it at least in theory easier to refactor the code to separate modules later and also makes it easier to follow the data flow.